src/setup-jest.ts

```diff
@@ -92,6 +93,7 @@ defineGlobalsInjections({
     AppLoaderModule,
   ],
   providers: [
+    mockAuth(),
```

Comment:
I don't know if mocking authentication globally is a good approach, there are places where this may not be desirable. Let's let developers mockAuth individually in their tests.